app/_components/form/Checkbox.tsx

```diff
@@ -9,7 +9,7 @@ interface Props {
 
 const Checkbox = ({ register, label, checked }: Props) => (
   <ChakraCheckbox {...register} defaultChecked={checked}>
-    {label}
+    <Text fontSize="sm">{label}</Text>
```

Comment:
I updated the styles here as well as in the CheckboxGroup, Radio, RadioGroup files to adjust these inputs so that their height was the same as other inputs in our forms. This involved reducing the font size a bit. Here's a pic...

![Selection_023](https://github.com/ed-pilots-network/frontend/assets/67390534/fdfc137d-1d6c-462e-96ed-a62ccabe8233)